### PR TITLE
Moved imgs in images folder to assets/images folder

### PR DESCRIPTION
**Pull request by jonathanzhang53**
https://docs.google.com/document/d/1duHrX_y2rqrcW0WcUO4ayrnoxq--gJp5gN7mWGaq_Vc/edit?usp=sharing

Summary of edits above in doc.

### CODE REVIEW

**Comment by jonathanzhang53** [comment]
GCI Task. My commits should be final now.

**Comment by tmccormi** [comment]
@muarachmann @Trodrige you guys are the GCI Mentors ... so this task is yours to review and test

**Comment by muarachmann** [comment]
pulling them

File: library/js/jquery.treeview-1.4.1/demo/index.html [hunk 1]
```diff
@@ -253,8 +253,8 @@ <h4>Sample 3 - fast animations, all branches collapsed at first, red theme, cook
 	
 	<h4>Sample 4 - two trees with one tree control, black and gray theme, cookie-based persistance</h4>
 	<div id="treecontrol">
-		<a title="Collapse the entire tree below" href="#"><img src="../images/minus.gif" /> Collapse All</a>
-		<a title="Expand the entire tree below" href="#"><img src="../images/plus.gif" /> Expand All</a>
+		<a title="Collapse the entire tree below" href="#"><img src="../../../../assets/images/minus.gif" />Collapse All</a>
```
Comment (aethelwulffe) [inline library/js/jquery.treeview-1.4.1/demo/index.html]: You sure this is correct?  I can check.  Mind you, this DEMO file is junk.  Just making sure this is not a blind find and replace.  There are a LOT of features I need to look at to test this PR.

File: patient_portal/messaging/secure_chat.php [hunk 2]
```diff
@@ -975,10 +975,10 @@ public function pingAction()
                                         ng-class="{'pull-left':!message.me, 'pull-right':message.me}">{{message.date }}</span>
                                 </div>
                                 <img class="direct-chat-img" ng-show="!message.me"
-                                    src="<?php echo $GLOBALS['images_path']; ?>/Unknown-person.gif"
+                                    src="<?php echo $GLOBALS['images_path']; ?>/unknown-person.gif"
```
Comment (aethelwulffe) [inline patient_portal/messaging/secure_chat.php]: Case sensitive check needed here.  Make sure the file name resolves on Linux server (it still would on Windows).

File: patient_portal/messaging/secure_chat.php [hunk 3]
```diff
@@ -975,10 +975,10 @@ public function pingAction()
...
                                     alt="">
                                 <img class="direct-chat-img" ng-show="message.me")
-                                    src="<?php echo $GLOBALS['images_path']; ?>/favicon-32x32.png"
+                                    src="<?php echo $GLOBALS['images_path']; ?>lhcross.png"
```
Comment (aethelwulffe) [inline patient_portal/messaging/secure_chat.php]: Is the images_path index the one that points to /assets/images?

File: templates/documents/general_queue.php [hunk 4]
```diff
@@ -49,7 +49,7 @@ function submit_documents()
 
         <td><a href="javascript:" onclick="top.restoreSession();
             var URL='controller.php?patient_finder&find&form_id=queue<?php echo "['files[`$file.document_id`][patient_id]']";?>&form_name=patient_name'; 
-            window.open(URL, 'queue', 'toolbar=0,scrollbars=1,location=0,statusbar=1,menubar=0,resizable=1,width=450,height=400,left = 425,top = 250');"><img src="images/stock_search-16.png" border="0"></a>&nbsp;&nbsp;&nbsp;</td>
+            window.open(URL, 'queue', 'toolbar=0,scrollbars=1,location=0,statusbar=1,menubar=0,resizable=1,width=450,height=400,left = 425,top = 250');"><img src="assets/images/stock_search.png" border="0"></a>&nbsp;&nbsp;&nbsp;</td>
```
Comment (aethelwulffe) [inline templates/documents/general_queue.php]: This does not work.  Broken.

File: templates/documents/general_view.php [hunk 5]
```diff
@@ -199,7 +199,7 @@ function import_ccr(docid)  {
             <a href="javascript:"
             onclick="top.restoreSession();
             var URL='controller.php?patient_finder&find&form_id={"document_move['new_patient_id']"|escape:"url"}&form_name={"document_move['new_patient_name']"|escape:"url"}'; window.open(URL, 'document_move', 'toolbar=0,scrollbars=1,location=0,statusbar=1,menubar=0,resizable=1,width=450,height=400,left=425,top=250');">
-            <img src="images/stock_search-16.png" border="0" /></a>
```
Comment (aethelwulffe) [inline templates/documents/general_view.php]: Broken.

**Comment by aethelwulffe** [comment]
This is a good housekeeping PR, but there were some directory name issues (like with capitalization of directory names) that can/would break things.  Also, there were some demo files in assets that made me worry that a general find-and-replace was done vs. careful unit test on each. @jonathanzhang53 can you please answer some of the PR review questions?

**Comment by aethelwulffe** [comment]
Also, file images in /modules don't get moved to a general assets directory.  They belong to the module, which should be treated as a monolithic package.

**Comment by jonathanzhang53** [comment]
@aethelwulffe Yeah I'll admit that I wasn't really sure of some of the stuff I was doing while trying to move some of these images because there was some file stuff that I didn't understand. I'll take a look once the school year is out and have some time.

**Pull request closed without merging**